### DISCUSSION TITLE
Can I add more exported variables to .bashrc and not use a config file?

### DISCUSSION THREAD

**Discussion by David-Else** [pystardust/ytfzf#507] [Q&A] [answered]
I have `export YTFZF_PREF="bestvideo[height<=?2160]+bestaudio/best"`in the `.bashrc`.

I want to add `thumbnail_viewer` to be kitty. All the docs talk about a config file (I don't want to make). Can I add this to the `.bashrc` somehow? What is the syntax? Thanks

**Comment by Euro20179** [comment, marked as answer]
technically theres no support for exporting, but i suppose

`export thumbnail_viewer=kitty` should work

also as of 2.0 use `video_pref` as opposed to `YTFZF_PREF` 

Another solution would be to have a wrapper script that looks somethibg like this


```sh
#!/usr/share/local/bin/ytfzf

thumbnail_viewer=kitty
video_pref=...

```

replace the shebang to the path to ytfzf

<del>
edit:

you could also have a function in .bashrc that looks something like

```sh
ytfzf() {
    video_pref="..." command ytfzf --thumb-viewer=kitty "$@"
}
```

im not 100℅ sure this works since i havemt tested it, but you can try

</del>

This does not work

**Reply by David-Else** [reply]
Works great, thanks!
